content/schedule.md

```diff
@@ -8,10 +8,9 @@ weight: 1
 
 The schedule of the day itself is still uncertain. The schedule of this year looks something like this:
 
-| Month     | Action                                                    |
-| --------- | --------------------------------------------------------- |
-| June      | Announcement date of the BAPC preliminaries               |
-| September | BAPC preliminaries (organized by individual universities) |
-| October   | BAPC 2021                                                 |
+| Date       | Action                                                    |
+| ---------- | --------------------------------------------------------- |
+| 9 October  | BAPC preliminaries (organized by individual universities) |
```

Comment:
Not all sites might pick that date.

Reply:
I think given the disclaimer at the bottom allows us wiggle room. + we can add any additional deviations either as an asterisk later on or by adding another date to this right (since atm you haven't heard anyone who's not doing it on this date?)

Reply:
I did but that's not important for this schedule.